### PR TITLE
fix: only load JSON files for localization

### PR DESCRIPTION
**Pull request by ecmadao**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -59,6 +59,9 @@ function mergedLocalMessages(folder) {
   const message = {};
   const pathes = fse.readdirSync(folder);
   for (const name of pathes) {
+    if (name[0] === ".") {
```
Comment (boojack) [inline nuxt.config.js:62]:
```suggestion
    if (name.startWith(".")) {
```

**Review by boojack — APPROVED** [review]
LGTM

File: nuxt.config.js (hunk 1 above)
Comment (boojack) [reply nuxt.config.js:62]: SGTM

**Review by boojack — DISMISSED** [review]
LGTM

File: nuxt.config.js (hunk 1 above)
Comment (tianzhou) [inline nuxt.config.js:62]: let's add an inline comment as well

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by ecmadao** [comment]
I'd like to use another solution: only load JSON files.
Code updated, PTAL @tianzhou @boojack

**Review by tianzhou — APPROVED** [review]
LGTM

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**